src/core/plugins/react-17/index.js

```diff
@@ -0,0 +1,14 @@
+import { render } from "./root-injects"
```

Comment:
Let's inject prettier comment so that the formatting of code is handled by prettier.

```suggestion
/**
 * @prettier
 */
import { render } from "./root-injects"
```